scripts/actions/build-deb/build-deb.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
Most of this is the same as https://github.com/yarnpkg/yarn/blob/master/scripts/build-deb.sh